### PR TITLE
Feature/reset animations

### LINKED ISSUE #38: Add an option to choose if on stop, elements should reset to their original position or not
Currently, if you stop, it will stop the music and the element will not reset to their position.
I feel that the user should be able to choose the behavior.
I consider adding this into the library itself

This would required some changes like :
 - A way to provide the option or a "stopAndReset" function
 - Each dance would need a reset function that could be called to reset elements.

### PR DESCRIPTION
**Pull request by nesvand**
Resolves #38 by adding a flag to `stop`

### CODE REVIEW

File: src/rythm.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default class Rythm {
     this.addRythm('rythm-bass', 'pulse', 0, 10)
     this.addRythm('rythm-medium', 'pulse', 150, 40)
     this.addRythm('rythm-high', 'pulse', 400, 200)
+    this.animationFrameRequest = void 0
```
Comment (Okazari) [inline src/rythm.js]: What is the purpose of this line ?

Comment (nesvand) [reply src/rythm.js]: defaulting to `undefined` on load so we don't end up attempting to cancel an animation that doesn't exist ;)

Comment (Okazari) [reply src/rythm.js]: Animation cancel were already done in line 47, but i guess this is cleaner to cancel the animation request :+1:

Comment (nesvand) [reply src/rythm.js]: I completely forgot to save the request each frame - my bad - fixed ;)

**Pull request merged**